test/set_sharding_metadata_test.lua

```diff
@@ -154,8 +154,11 @@ function g.test_two_sharding_spaces()
     space_two.sharding_key = {
         'unsigned_nonnull', 'integer_nonnull', 'string_nonnull'
     }
+    local some_module = {
```

Comment:
To be honest I don't understand how it could fix anything? Please describe.

Reply:
Ok. Thanks for explanation.

But if there is the same code in different test you can simply move this logic to before_tests hook

Reply:
Ok. Anyway lgtm.